### PR TITLE
Add profile to conditionally build account2 theme

### LINKED ISSUE #10266: Allow Account V2 theme to be built conditionally
The build of Account V2  theme can take up a lot of time as it requires Node to be installed and NPM to run various tasks. It would be useful to be able to disable this part so that if it's not needed CI of other projects can pass.

Specifically we want this so we can speed up the time it takes for the integration tests to start running for the new [Admin UI](github.com/keycloak/keycloak-admin-ui/).

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #10266

### CODE REVIEW

**Review by stianst — CHANGES_REQUESTED** [review]
I'm not okay with this approach and would like to discuss alternatives

File: themes/pom.xml [hunk 1]
```diff
@@ -131,6 +86,72 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>account2</id>
+            <activation>
+                <!-- 'activeByDefault' will not work here as other profiles in this POM are always active, so we need to have a workaround to activate this profile. -->
+                <file>
```
Comment (stianst) [inline themes/pom.xml:93]: Use properties to activate this like this:
https://github.com/keycloak/keycloak/blob/bbe9ab38bc6afeaac95c995efd0e081aeb9b0e06/pom.xml#L2038

Comment (stianst) [reply themes/pom.xml:93]: This approach is consistent with what we do elsewhere and also useful for Devs in this repo to disable account console build if they want to save some time.

File: themes/pom.xml [hunk 2]
```diff
@@ -131,6 +86,72 @@
...
+                    <exists>.</exists>
+                </file>
+           </activation>
```
Comment (stianst) [inline themes/pom.xml:96]:
```suggestion
            <activation>
                <property>
                    <name>!skipAccount2</name>
                </property>
           </activation>
```

Comment (stianst) [reply themes/pom.xml:96]: This allows de-activating account2 generation with:

    mvn install -DskipAccount2

Comment (jonkoops) [reply themes/pom.xml]: Awesome, I really like this approach. Added your suggestion :+1:

**Review by ssilvert — APPROVED** [review]
LGTM

**Review by stianst — APPROVED** [review]

**Pull request merged**